internal/driver/gl/window.go

```diff
@@ -990,6 +998,16 @@ func (w *window) runWithContext(f func()) {
 	glfw.DetachCurrentContext()
 }
 
+func (w *window) dispatchSerialEvent(fn func()) {
+	w.serialEvents <- fn
+}
+
+func (w *window) runSerialEvents() {
+	for fn := range w.serialEvents {
+		fn()
```

Comment:
should fn() be run on main ?   Im not sure, but I think that KB events may require running on main.  

I need to have a play with that.

Reply:
We keep all callbacks off main. The “keyboard handling” that results from these callbacks is just go code updating. All the input handling is already done and/or calling back to main when the driver needs to

Reply:
Agreed. Since they didn't run on main before this PR, this won't make things any worse.